### PR TITLE
chore(deps): change schedule of dependency updates check

### PR DESCRIPTION
**Pull request by mpro7**
### Task Description/Number

This PR changes the schedule of dependency updates check from weekly to twice in a month.

Issue Number: DEV-

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] Bugfix: represents bug fixes
- [ ] Refactor: represents production code refactoring
- [ ] Feature: represents a new feature
- [ ] Documentation: documentation changes (no production code change)
- [x] Chore: maintenance tasks (no production code change)
- [ ] Style: styles updates (no production code change)
- [ ] Test: all about tests: adding, refactoring tests (no production code change)
- [ ] Other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [x] No

### CODE REVIEW

File: .github/workflows/scala-steward.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Dependencies
 # This workflow will launch at 00:00 every Sunday
```
Comment (irinaschubert) [inline .github/workflows/scala-steward.yml:3]:
```suggestion
# This workflow will launch at 00:00 on every 1th and 15th of the month
```

**Review by irinaschubert — CHANGES_REQUESTED** [review]
Are we notified if a security critical upate is pending or how do we deal with that?

**Comment by mpro7** [comment]
> Are we notified if a security critical upate is pending or how do we deal with that?

They are listed here. Notification should be send via email or whatever the setup is.
https://github.com/dasch-swiss/dsp-api/security/dependabot

**Comment by irinaschubert** [comment]
> > Are we notified if a security critical upate is pending or how do we deal with that?
> 
> They are listed here. Notification should be send via email or whatever the setup is. https://github.com/dasch-swiss/dsp-api/security/dependabot

I would make sure that this setting is activated, otherwise we might miss important updates. Could you do that?

**Comment by mpro7** [comment]
> > > Are we notified if a security critical upate is pending or how do we deal with that?
> > 
> > 
> > They are listed here. Notification should be send via email or whatever the setup is. https://github.com/dasch-swiss/dsp-api/security/dependabot
> 
> I would make sure that this setting is activated, otherwise we might miss important updates. Could you do that?

This isn't related to this PR, however I did check
> By default, we notify people with admin permissions in the affected repositories about new Dependabot alerts.

**Review by irinaschubert — APPROVED** [review]

**Pull request merged**